sos/policies/arch.py

```diff
@@ -0,0 +1,48 @@
+from sos.policies import PackageManager, LinuxPolicy
+from sos.plugins import ArchPlugin
+from sos.utilities import shell_out
+
+
+class ArchPolicy(LinuxPolicy):
+
+    distro = "Arch Linux"
+    vendor = "Arch Linux"
+    vendor_url = "https://www.archlinux.org/"
+    vendor_text = ""
+    # package_manager = PackageManager(
+    #   "pacman --query | awk 'BEGIN {OFS = \"|\"} {print $1,$2}'")
+    valid_subclasses = [ArchPlugin]
+
+    def __init__(self, sysroot=None):
+        super(LinuxPolicy, self).__init__(sysroot=sysroot)
```

Comment:
there should be `super(ArchPolicy, self)....` to call `__init__` method of parent class of *this* `ArchPolicy` class (i.e. to call the method of `LinuxPolicy` - not parent class of `LinuxPolicy` that is class `Policy` ;-)